apds9960/apds9960_nano_33_ble.go

```diff
@@ -0,0 +1,28 @@
+// +build nano_33_ble
+
+package apds9960
+
+import (
+	"machine"
+	"time"
+
+	"tinygo.org/x/drivers"
+)
+
+// New creates a new APDS-9960 connection. The I2C bus must already be
+// configured.
+//
+// This function only creates the Device object, it does not touch the device.
+func New(bus drivers.I2C) Device {
+	// turn on internal power pin (machine.P0_22) and I2C1 pullups power pin (machine.P1_00)
+	// and wait a moment.
+	ENV := machine.P0_15
```

Comment:
I wonder, would it be good idea to extract these special pins to tinygo repo, in `board_nano-33-ble.go`?
Also, this initialisation sequence seem to be generic for this sensor, only difference from board to board is pins? If so, why not move it into `apds9960.Configure`? Much the same as wifinina driver does, for example.